components/Table/Table.js

```diff
@@ -95,12 +113,184 @@ export class Table {
     this.tableHeader.append(tableHeaderRow);
   }
 
+  renderDropDownMenu(menuBtn, isSortable, cell) {
+    const dropDownMenu = document.createElement('div');
+    dropDownMenu.classList.add('drop-down__menu');
+    this.menuDropDownItems = this.menuConfig
+      .filter((elConfig) => {
+        if (
+          this.callbacksMap.sortCallback ||
+          this.callbacksMap.filterCallback
+        ) {
+          const menuElSort =
+            elConfig.key === MenuElMap.SORT_BY_ASC ||
+            elConfig.key === MenuElMap.SORT_BY_DESC;
+          if (!isSortable && menuElSort) {
+            return;
```

Comment:
filter should return `true` or `false`
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Doesn't make sense to write in such a style
We can just write
`return !isSortable && menuElSort;`